### ISSUE TITLE
Exception in script.rpy.

### ISSUE THREAD

**Issue by renpytom** [renpy/renpy#2841]
```
-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/bootstrap.py", line 326, in bootstrap
    renpy.main.main()
  File "renpy/main.py", line 447, in main
    renpy.game.script.load_script() # sets renpy.game.script.
  File "renpy/script.py", line 283, in load_script
    self.load_appropriate_file(".rpyc", ".rpy", dir, fn, initcode)
  File "renpy/script.py", line 780, in load_appropriate_file
    if data is None:
UnboundLocalError: local variable 'data' referenced before assignment

Windows-10-10.0.19041
Ren'Py 7.4.4.1439
```

**Comment by KiloOscarSix** [comment]
- Using Github in an organisation of ~20 people
- We use the following .gitignore to simplify Github layout:
 `.vscode/
lib/
renpy/

game/bugTesting/_bugTesting.txt
game/cache/
game/saves/

CollegeKings.py
CollegeKings.sh
errors.txt
gradle.properties.txt
log.txt
Stored Script.txt
traceback.txt
project.json
steam_appid.txt

screenshot*

*.bat
*.rpa
*.rpyc
*.zip
*.exe
*.bak
`
- The ignoring of `.rpyc` between branches is our current speculation of the issue
- Reloading the game has solved this issue on the occasions I have catched it

**Comment by renpytom** [comment]
e37615b0b should at least report a better error.

**Comment by renpytom** [comment]
It's been released in 7.4.6.
